### DISCUSSION TITLE
Need help for connecting UG15 PLC

### DISCUSSION THREAD

**Discussion by Ponganes** [artisan-roaster-scope/artisan#929] [Q&A] [answered]
<img width="1208" alt="Screenshot 2565-08-03 at 16 29 15" src="https://user-images.githubusercontent.com/29407807/182576248-c68f438c-d4ba-4d18-91a7-a56fe14ac72c.png">

Hi everyone,

I use to connect really smooth my machine with Artisan, after changed the PC to I Mac, I tried to do the same configuration with the IP address, Subnet mask but It doesn't work as photo show the error " S7 Communication error readInt(5,200,72,False): Result None@line755".
Hope someone know how to fix i.

Best regards,
Rawi

**Comment by MAKOMO** [comment, marked as answer]
Seems that you configured Artisan for an UG with the PLC control functionality. Do your machine is running this updated PLC firmware which allows to control the machine? Maybe your PLC is running the previous firmware which does not support control. In that case you should configure for the "Probat UG" machine setup without the "Control" tag.

Is the configuration in the S7 tab under menu Config >> Ports (4th tab) equal under both configurations?

**Reply by Ponganes** [reply]
Hi Mate,
Your suggestions are appreciated.
Simply fixed  just configuration without "Control" :)

Best regards,
Rawi
